care/facility/api/serializers/patient_external_test.py

```diff
@@ -91,6 +97,14 @@ def validate_empty_values(self, data, *args, **kwargs):
 
         return super().validate_empty_values(data, *args, **kwargs)
 
+    def validate(self, attrs):
+        user = self.context["request"].user
+        if user.user_type < User.TYPE_VALUE_MAP["Nurse"]:
+            raise ValidationError(
+                {"user": ["User is not allowed to perform this action"]}
+            )
+        return super().validate(attrs)
```

Comment:
Same, Why are Authz checks done in the serializer validate? Shouldn't this be done by DRY Permissions?